pkg/cmd/pr/shared/finder.go

```diff
@@ -250,21 +254,23 @@ func (f *finder) parseCurrentBranch() (string, int, error) {
 	}
 
 	var gitRemoteRepo ghrepo.Interface
-	if branchConfig.RemoteURL != nil {
+	if branchConfig.PushRemoteURL != nil {
 		// the branch merges from a remote specified by URL
 		if r, err := ghrepo.FromURL(branchConfig.RemoteURL); err == nil {
```

Comment:
Looks like this should be `ghrepo.FromURL(branchConfig.PushRemoteURL)`, though I think I'd advocate for the URL and Name to be collapsed to a single field string, and we disambiguate URL vs Name here rather than in the `ReadBranchConfig`.